README.md

```diff
@@ -57,11 +57,15 @@ $ ./bin/nemo --help
 
 ### `output`
 
-### `output.reports`
+### `output.reports <optional>`
+
+This convenience setting will create timestamped and tag-based directory structure for reports and
+screenshots when you use [mochawesome](https://www.npmjs.com/package/mochawesome) or [xunit](https://mochajs.org/#xunit) reporters. When you use this, you can omit the specific
+directory/filename settings for those reporters, as nemo will take care of that for you.
```

Comment:
What are the benefits of `mochawesome` and `xunit` respectively? I'm hoping that the `nemo-reporter` I'm integrating can be a default (but not a mandate) and can save teams from setting up 2+ reporters for different environments. So far, it's worked better for the people that I've onboarded around consumer than those tools both locally and on CI.

I don't know a lot about those tools, though. If there are good reasons to keep encouraging them, ideally I would just add those good reasons/features to nemo-reporter and the docs can be made more simple (where a new user doesn't have to worry about setting up a reporter at all).

Reply:
I agree. Do you have an ETA on being able to get the reporter out in open source?

Reply:
I'll have a PR tonight, I want to try to marry it from this repo. I want to see how that feels. We can go from there if it feels right.